LearningHub.Nhs.UserProfileUI/Services/DigitalStaffPassportService.cs

```diff
@@ -292,7 +295,8 @@ private Dictionary<string, string> PopulateCertificateClaims(VerifiableCredentia
             ////claims.Add($"{verifiableCredential.ClaimPrefix}-Pedigree", "notset");
 
             // TODO - this hard-coded value needs to be picked up from the user's specific identity value
-            claims.Add("UniqueIdentifier", "3ed5a980-3a1b-45a0-9c50-6a58127cb19f");
+           // claims.Add("UniqueIdentifier", "3ed5a980-3a1b-45a0-9c50-6a58127cb19f");
```

Comment:
Do we still need this commented line of code and the TODO?

Reply:
I cherry-picked my commit after updating the remote origin on my local PC. Obviously this approach seems problematic. I'll rework this PR